common/djangoapps/util/date_utils.py

```diff
@@ -89,7 +103,7 @@ def from_timestamp(timestamp):
     If the timestamp cannot be converted, returns None instead.
     """
     try:
-        return datetime.utcfromtimestamp(int(timestamp)).replace(tzinfo=UTC)
+        return datetime.utcfromtimestamp(int(timestamp)).replace(tzinfo=utc)
```

Comment:
@jcdyer correct me if i'm wrong. `UTC` wasn't included in the list of attributes/functions you should call, but `utc` was.